content/docs/user-guide/managing-external-data.md

```diff
@@ -19,12 +19,13 @@ outputs for `dvc.yaml` files (only `outs` field, not metrics or plots).
 External outputs are considered part of the (extended) DVC project: DVC will
 track changes in them, and reflect this in `dvc status` reports, for example.
 
-For cached external outputs (e.g. `dvc add`, `dvc run -o`), you will need to
-[setup an external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
-in the same external/remote file system first.
+Normally, external outputs (e.g. `dvc add`, `dvc run -o`), require an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+in the same external/remote file. The exception is when they are not
+<abbr>cached</abbr> (`--no-commit` option).
```

Comment:
not cached and no-commit are different things (like -O and --no-commit for regular outputs) Also, as I checked `--no-commit` still requires cache to be set.

Reply:
OK... I removed the last sentence for now. It's kind of confusing/hard to explain right. Maybe we should review the behavior in the core repo first?

Reply:
`--no-commit` workflow is not about caching/not caching. Usually it is about deliberately _delaying_ the caching for some reason.

Reply:
OK. Is that relevant to mention especially in this guide? Seems like a general thing for add/run (and it's already mentioned in those refs).

The reason it was mentioned here is that it is the exception to the "Normally" at the beginning of the paragraph.

Reply:
Yep, I would avoid changing it to `Normally, ` and then we don't need mentioning this.

Reply:
OK, I'm removing "Normally", but then this isn't always true. But maybe it will be fixed by the changes per our other discussion here ⌛

Reply:
They do always require the external cache as we checked, right? Also, I don't see a good use case w/o cache.

Reply:
Yes they do. Which seems like a bug to me, but if there's no use case for the combination anyway then I guess this is not an issue. Resolving!